installers/ad-synchrona-14-leds.sh

```diff
@@ -17,6 +17,24 @@ AD9545_LOCKED=0
 HMC7044_LOCKED=0
 AD9545_FREERUN=0
 
+trap synchrona_leds_exit SIGTERM
+function synchrona_leds_exit()
+{
+	echo $RED_LED1 > "/sys/class/gpio/unexport"
+	echo $GREEN_LED1 > "/sys/class/gpio/unexport"
+	exit
+}
+
+if [ ! -d "/sys/class/gpio/gpio$RED_LED1" ]
+then
+	echo $RED_LED1 > "/sys/class/gpio/export"
+fi
```

Comment:
nit: this could be a one liner `[ ! -d "/sys/class/gpio/gpio$RED_LED1" ] && echo $RED_LED1 > "/sys/class/gpio/export"`